packages/nextjs/src/config/webpack.ts

```diff
@@ -195,22 +159,20 @@ function addFileToExistingEntryPoint(
   const currentEntryPoint = entryProperty[entryPointName];
   let newEntryPoint: EntryPointValue;
 
-  // We inject the user's client config file after the existing code so that the config file has access to
-  // `publicRuntimeConfig`. See https://github.com/getsentry/sentry-javascript/issues/3485
   if (typeof currentEntryPoint === 'string') {
-    newEntryPoint = [currentEntryPoint, filepath];
+    newEntryPoint = [filepath, currentEntryPoint];
   } else if (Array.isArray(currentEntryPoint)) {
-    newEntryPoint = [...currentEntryPoint, filepath];
+    newEntryPoint = [filepath, ...currentEntryPoint];
```

Comment:
Why is filepath in the front of the array now?

Reply:
> By making sure to add it specifically to the beginning of each API entry point, we can ensure that the SDK will be up and running by the time any code in the route file runs.

LMK if this isn't clear or you have more questions.

Reply:
(Also, FWIW, it actually _won't_ work to add it afterwards, because it confuses the heck out of nextjs, which ends up complaining that `_app` doesn't export a legit react component. Even though the code we're adding doesn't `export default` anywhere - either in the user's code or the SDK which it pulls in with it - I suspect that in the transpiling process something gets turned into an `export default`,  and putting all that code second overwrites the `export default App` that next is looking for.)